app/views/admin/statistics_announcement_publications/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :page_title, "Connect document" %>
+<% content_for :context, @statistics_announcement.title %>
+<% content_for :title, "Connect document" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @statistics_announcement)) %>
+
+<%= form_with url: admin_statistics_announcement_publication_index_path(@statistics_announcement), method: :get do |_form| %>
+  <%= render "govuk_publishing_components/components/search", {
+    name: "search",
```

Comment:
If we do this then this would need be updated to `title` https://github.com/alphagov/whitehall/pull/7660/files#r1195196818

Reply:
✅